JS-Algorithms/challenges.js

```diff
@@ -41,7 +42,11 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+   let arr = [...COUNTRY_NAMES];
+   for (let i = 0; i <= arr.length - 1 + index; i++) {
+      arr.push(arr.shift());
+   }
```

Comment:
Prefer the use of `forEach` instead of a traditional `for`. And if you are dealing with async/await use `for of`

Reply:
Thank you so much I would consider to improve my coding